### ISSUE TITLE
yang paths with collapsed prefixes?

### ISSUE THREAD

**Issue by hellt** [karimra/gnmic#502]
I wonder if it is possible to create a gnmic path/generate option that would collapse the prefixes of a given path.

Consider an example of a path that uses module prefixes:

```
/srl_nokia-netinst:network-instance[srl_nokia-netinst:name=*]/srl_nokia-netinst:protocols/srl_nokia-bgp:bgp/srl_nokia-bgp:evpn/srl_nokia-bgp:active-routes
``` 

Each element of the path comes with a prefix, although YANG allows using module prefixes only when it is changed from the parent node; for example, the valid notation of this path:

```
/srl_nokia-netinst:network-instance[name=*]/protocols/srl_nokia-bgp:bgp/evpn/active-routes
``` 

The logic of using a path for a given element is as follows:
1. the first element comes with a prefix always
2. next element should be prefixed if its prefix is not equal to the prefix of its parent

**Comment by karimra** [comment]
This should be possible, I will try to implement it

**Comment by karimra** [comment]
released as part of v0.21.0
